static/app/components/events/featureFlags/useGroupFlags.tsx

```diff
@@ -0,0 +1,37 @@
+import {defined} from 'sentry/utils';
+import {type ApiQueryKey, useApiQuery} from 'sentry/utils/queryClient';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {
+  FetchIssueTagsParameters,
+  GroupTag,
+  GroupTagUseQueryOptions,
+} from 'sentry/views/issueDetails/groupTags/useGroupTags';
+
+const makeGroupFlagsQueryKey = ({
+  groupId,
+  orgSlug,
+  environment,
+  readable,
+  limit,
+}: FetchIssueTagsParameters): ApiQueryKey => [
+  `/organizations/${orgSlug}/issues/${groupId}/tags/`,
+  {query: {environment, readable, limit, useFlagsBackend: '1'}},
+];
+
+export default function useGroupFlags(
+  parameters: Omit<FetchIssueTagsParameters, 'orgSlug'>,
```

Comment:
doesn't feel like the parameters are all that similar to FetchIssueTagsParameters like does flags take a `readable`